bump-version/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Bump Version
+description: Update package.json with the new version number & Sets the version as ENV variable.
+
+inputs:
+  CHANNEL:
+    required: true
+
+outputs:
+  prev_version:
+    description: "previous cloud version"
+    value: ${{ steps.bump_version_step.outputs.prev_version }}
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      id: bump_version_step
+      run: |
+            npm install --no-package-lock --no-save semver@7.3.4
+            PREV_PACKAGE_VERSION=$(node -p "require('./package.json').last_cloud_version")
```

Comment:
Use the channel, `last_${ channel }}_version`

Reply:
done